changelog.md

```diff
@@ -15,7 +15,7 @@ cpprestsdk (2.10.18)
 
 cpprestsdk (2.10.17)
 * PR#1550 Fix year calculation for the last day of a leap year
-* PR#1523 Fix wrong linking of Apple Frameworks on MacOS
+* PR#1523 Fix wrong linking of Apple Frameworks on macOS

```

Comment:
Some projects don't like changing changelogs, I'm happy to drop this (or anything else) if it's a problem.